.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,22 +6,27 @@
 -->
 
 **What type of PR is this?**
-> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
->
-> /kind api-change
-> /kind bug
-> /kind cleanup
-> /kind design
-> /kind documentation
-> /kind test
-> /kind failing-test
-> /kind feature
+
+<!--
+Add one of the following kinds:
+/kind bug
+/kind cleanup
+/kind documentation
+/kind feature
+/kind test
+/kind design
+
+Optionally add one or more of the following kinds if applicable:
+/kind api-change
+/kind failing-test
+/kind regression
```

Comment:
now we still have no the `regression` kind label, need upgrade the prow later.

Reply:
Get it. Tracked as #2104